impeller/scene/importer/BUILD.gn

```diff
@@ -0,0 +1,93 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/impeller/tools/impeller.gni")
+import("//flutter/shell/version/version.gni")
+import("//third_party/flatbuffers/flatbuffers.gni")
+
+config("runtime_stage_config") {
+  configs = [ "//flutter/impeller:impeller_public_config" ]
+  include_dirs = [ "$root_gen_dir/flutter" ]
+}
+
+flatbuffers("importer_flatbuffers") {
+  flatbuffers = [ "mesh.fbs" ]
+  public_configs = [ ":runtime_stage_config" ]
+  public_deps = [ "//third_party/flatbuffers" ]
+}
+
+impeller_component("importer_lib") {
+  # Current versions of libcxx have deprecated some of the UTF-16 string
+  # conversion APIs.
+  defines = [ "_LIBCPP_DISABLE_DEPRECATION_WARNINGS" ]
+
+  sources = [
+    "importer.h",
+    "importer_gltf.cc",
+    "switches.cc",
+    "switches.h",
+    "types.h",
+  ]
+
+  public_deps = [
+    ":importer_flatbuffers",
+    "../../base",
+    "../../compiler:utilities",
+    "../../geometry",
+    "//flutter/fml",
+
+    # All third_party deps must be reflected below in the impellerc_license
+    # target.
+    # TODO(bdero): Fix tinygltf compilation warnings.
+    #"//third_party/tinygltf",
+  ]
+}
+
+generated_file("scene_importer_license") {
+  source_path = rebase_path(".", "//flutter")
+  git_url = "https://github.com/flutter/engine/tree/$engine_version"
+  outputs = [ "$target_gen_dir/LICENSE.impellerc.md" ]
+  contents = [
+    "# scene_importer",
+    "",
+    "This tool is used by the Flutter SDK to import 3D geometry.",
+    "",
+    "Source code for this tool: [flutter/engine/$source_path]($git_url/$source_path).",
+    "",
+    "## Licenses",
+    "",
+    "### scene_importer",
+    "",
+    read_file("//flutter/sky/packages/sky_engine/LICENSE", "string"),
+    "",
+
+    # These licenses are ignored by the main license checker, since they are not
+    # shipped to end-application binaries and only shipped as part of developer
+    # tooling in impellerc. Add them here.
+    "## Additional open source licenses",
+    "",
+    "### tinygltf",
+    "",
+    read_file("//third_party/tinygltf/LICENSE", "string"),
+  ]
+}
+
+group("importer") {
```

Comment:
Nit: On naming, perhaps just `scenec` like `impellerc`. `importer` is rather generic.

Reply:
Ah yeah, `scenec` seems good. My mind kept getting stuck on `geometryc` which I didn't like very much.